mimic/rest/maas_api.py

```diff
@@ -573,35 +568,41 @@ def overview(self, request, tenant_id):
         """
         serves the overview api call,returns all entities,checks and alarms
         """
-        entities = self._entity_cache_for_tenant(tenant_id).entities_list
-        checks = self._entity_cache_for_tenant(tenant_id).checks_list
-        alarms = self._entity_cache_for_tenant(tenant_id).alarms_list
-        metadata = {}
-        metadata['count'] = len(entities)
-        metadata['marker'] = None
-        metadata['next_marker'] = None
-        metadata['limit'] = 1000
-        metadata['next_href'] = None
-        values = []
-        for e in entities:
-            v = {}
-            v['alarms'] = []
-            for a in alarms:
-                if a['entity_id'] == e['id']:
-                    a = dict(a)
-                    del a['entity_id']
-                    v['alarms'].append(a)
-            v['checks'] = []
-            for c in checks:
-                if c['entity_id'] == e['id']:
-                    c = dict(c)
-                    del c['entity_id']
-                    v['checks'].append(c)
-            v['entity'] = e
-            v['latest_alarm_states'] = []
-            values.append(v)
-        request.setResponseCode(200)
-        return json.dumps({'metadata': metadata, 'values': values})
+        # This is a hack to make cloud control panel server details page finish loading the server
+        # details
+        if 'uri' in request.args:
+            request.setResponseCode(404)
+            return json.dumps({})
+        else:
```

Comment:
I have a couple of concerns here.

1. Why do we need to 404 in this case at all? Successfully getting the overview response shouldn't cause a page to hang. Is this a bug in Reach? and if so, can you post a gist or similar explaining it?
2. Since you return at the end of the if block, you don't need the else block and re-indentation. I'd prefer you not indent everything over again if you don't have to.